bowtie/_cli.py

```diff
@@ -755,14 +809,45 @@ def validate(
     return asyncio.run(_run(fail_fast=False, **kwargs, cases=[case]))
 
 
+class _MakeRemoteDataGetter(Protocol):
+    def __call__(
+        self,
+    ) -> Callable[[str], None | list[tuple[str, str | int]]]: ...
+
+
 @implementation_subcommand()  # type: ignore[reportArgumentType]
-@FORMAT
-async def info(implementations: Iterable[Implementation], format: _F):
+@click.option(
+    "--format",
+    "-f",
+    "format",
+    help="What format to use for the output",
+    default=lambda: "pretty" if sys.stdout.isatty() else "json",
+    show_default="pretty if stdout is a tty, otherwise JSON",
+    type=click.Choice(["json", "file", "pretty", "markdown"]),
+)
+@NOREMOTEDATA
+async def info(
+    implementations: Iterable[Implementation],
+    format: Literal["json", "file", "pretty", "markdown"],
```

Comment:
(You still have `file` here)

Reply:
Yeah sorry I was waiting for your response to make other changes along with this.